runtime/queries/amber/highlights.scm

```diff
@@ -0,0 +1,60 @@
+(comment) @comment
+
+[
+    "if"
+    "loop"
+    "for"
+    "return"
+    "fun"
+    "else"
+    "then"
+    "break"
+    "continue"
+    "and"
+    "or"
+    "not"
+    "let"
+    "pub"
+    "main"
+    "echo"
+    "exit"
+    "fun"
+    "import"
+    "from"
+    "as"
+    "in"
+    "fail"
+    "failed"
+    "silent"
+    "nameof"
+    "is"
+    "unsafe"
+    "trust"
+] @keyword
+
+; Literals
+(boolean) @constant.builtin.boolean
+(number) @constant.numeric
+(null) @constant.numeric
+(string) @string
+(status) @keyword
+(command) @string
+(handler) @keyword
+(block) @delimeter
```

Comment:
```suggestion
(block) @punctuation.delimiter
```

For our highlights queries we use these scopes: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

which I think are a little different than Zed's (but mostly the same). There's also a little typo there "delimeter=>delimiter". (Maybe that should be changed in the Zed queries.)